data/curated/enmoma/meta.yaml

```
@@ -0,0 +1,11 @@
+title: English Monarchs and Marriages | Names, Ages, and Historical Years
```

```suggestion
title: English Monarchs and Marriages
```
The extra-long title might trip up social media posts, and I'll need to specifically watch for (and/or better deal with) `|` characters! That would break the tables of datasets.

---

Addressed in #726 (checks to deal with special characters) and #727 (improve instructions about title).